build.gradle

```diff
@@ -45,9 +54,9 @@ java {
 
 project.ext.excludesFromTestCoverage = ['**/DetectDownloadStrategy.java', '**/DetectPipelineStep.java', '**/DetectPostBuildStep.java', '**/DetectAirGapInstallation.java']
 
-group = 'com.blackducksoftware.integration'
-version = '9.0.0-SNAPSHOT'
-description = 'Blackduck Detect Plugin for Jenkins'
+group = 'com.blackduck.integration'
```

Comment:
@devmehtasynopsys Keep com.blackducksoftware.integration as group for this, since that is the namespace we will continue to use on actual Jenkins repo

Reply:
Okay I see will make that change.